modules/core/src/test/java/org/apache/synapse/config/xml/endpoints/DynamicLoadBalanceEndpointSerializerTest.java

```diff
@@ -0,0 +1,38 @@
+/*
+ *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.apache.synapse.config.xml.endpoints;
+
+import org.apache.axiom.om.OMElement;
+import org.apache.axiom.om.util.AXIOMUtil;
+import org.apache.synapse.SynapseException;
+import org.apache.synapse.endpoints.DynamicLoadbalanceEndpoint;
+import org.junit.Test;
+
+public class DynamicLoadBalanceEndpointSerializerTest {
+    @Test(expected = SynapseException.class)
+    public void test() throws Exception {
+        String inputXml = "<endpoint  xmlns=\"http://ws.apache.org/ns/synapse\">"
+                + "<dynamicLoadbalance policy=\"roundRobin\" failover=\"true\">"
+                + "    <membershipHandler class=\"org.apache.synapse.core.LoadBalanceMembershipHandler\">"
+                + "      <property name=\"name\" value=\"value\"/>" + "    </membershipHandler>"
+                + "  </dynamicLoadbalance>" + "</endpoint>";
+        OMElement inputElement = AXIOMUtil.stringToOM(inputXml);
+        DynamicLoadbalanceEndpoint endpoint = (DynamicLoadbalanceEndpoint) DynamicLoadbalanceEndpointFactory
+                .getEndpointFromElement(inputElement, true, null);
+        OMElement serializedResponse = DynamicLoadbalanceEndpointSerializer.getElementFromEndpoint(endpoint);
```

Comment:
What do you actually assert here?

Reply:
I don't assert anything. I check whether the exception is thrown.